### ISSUE TITLE
how to configure read,write,lock,hidden in specific folder

### ISSUE THREAD

**Issue by hayes-seyah09354085977** [Studio-42/elFinder#2581]
example 
main folder (root)
         - folder 1 (read only)
         - folder 2 (read and write)
         - folder 3 (locked)
         - folder 4 (hidden)

is that possible? i benchmark the windows explorer security 
![1-256](https://user-images.githubusercontent.com/35753309/42671475-bae9245c-8692-11e8-9256-8bb66086486f.png)

**Comment by nao-pon** [comment]
@CodeLyokoXtEAM The idea of hiding any item at the user level is interesting. However, it is unknown how much it is worth in elFinder. Can you explain the value?

**Comment by CodeLyokoXtEAM** [comment]
imagine users have permissions to access .tmb folder(or any other folder) in volume for cleanup it.
But it isn't necessary to show always.
So default settings for that folder is hidden & user can un-hide & access it when he want to cleanup it.
the hidden levels are just suggestion but when think about it, it unnecessary to elfinder.

**Comment by nao-pon** [comment]
@CodeLyokoXtEAM Okey, I made a new issue #2595.

Thanks! 👍

**Comment by nao-pon** [comment]
@hayes-seyah09354085977 Did you solved it?

**Comment by hayes-seyah09354085977** [comment]
yes sir i already solve that and i thinking of how to use that to separate file inside like Windows does
btw thanks for the support i love this elfinder :D
